### PR TITLE
Added Maps on Home Page

### PR DESCRIPTION
**Pull request by Akhigbepeace**
- Added the map on the home page of mobile and desktop screen sizes

image
image

- Started working on partner selector for mobile on the `Partners` page

### CODE REVIEW

**Review by chineduknight — APPROVED** [review]

File: src/Pages/Home/Index.jsx [hunk 1]
```diff
@@ -8,9 +8,35 @@ import ProjectDesc from "./ProjectDesc";
...
+          zIndex="-5"
+          w="700px"
+          h="800px"
+          objectFit="cover"
+          display={{
+            sm: "none",
+            md: "none",
+            lg: "none",
+            xl: "block",
+            "2xl": "block",
```
Comment (chineduknight) [inline src/Pages/Home/Index.jsx:36]: You don't need to do this like this you can just do the following 

```
 sm: "none",
xl: "block",
```

File: src/Pages/Home/Showcase/Map.jsx [hunk 2]
```diff
@@ -1,8 +1,27 @@
...
 
 const Map = () => {
-  return <Box>Map</Box>;
+  return (
+    <Box
+      mt={{
+        sm: "40px",
+        md: "40px",
+        lg: "40px",
+      }}
```
Comment (chineduknight) [inline src/Pages/Home/Showcase/Map.jsx:11]: This is unnecessary if all the values are the same then you don't need the responsiveness

File: src/Pages/Home/Showcase/Map.jsx [hunk 3]
```diff
@@ -1,8 +1,27 @@
...
+      <Image
+        src="https://res.cloudinary.com/wavecrest1/image/upload/v1669113391/wetland/Group_67_algkgk.png"
+        display={{
+          sm: "block",
+          md: "block",
+          lg: "block",
+          xl: "none",
+          "2xl": "none",
+        }}
+      />
```
Comment (chineduknight) [inline src/Pages/Home/Showcase/Map.jsx:22]: same here

**Review by chineduknight — APPROVED** [review]
Please correct these changes and make another PR

File: src/Pages/Home/Index.jsx (hunk 1 above)
Comment (Akhigbepeace) [reply src/Pages/Home/Index.jsx:36]: Okay sir...

**Pull request merged**